app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ItemAdapter2.kt

```diff
@@ -53,7 +52,7 @@ class ItemAdapter2(private var context: Context, var view_layout: Int, private v
         author.text = if (items.author[0] == "null") "Author not provided" else items.author[0]
         newsContent.text = items.description[0]
 
-        var date = ""
+        val date: String
```

Comment:
It was being redundant so just name it as String instead

Reply:
One change here:
The old way, it would be "", never null.
As it is here, it could be null.

I'd leave it the old way.